src/main/java/com/google/api/codegen/transformer/OutputTransformer.java

```diff
@@ -35,11 +36,33 @@
 import java.util.Set;
 import javax.annotation.Nullable;
 
-class OutputTransformer {
+public class OutputTransformer {
   private static final String RESPONSE_PLACEHOLDER = "$resp";
   private static final Set<String> RESERVED_KEYWORDS =
       ImmutableSet.<String>of("response", "response_item");
 
+  private final OutputImportTransformer importTransformer;
+
+  public OutputTransformer() {
+    this.importTransformer = new OutputImportTransformer() {};
+  }
+
+  public OutputTransformer(OutputImportTransformer importTransformer) {
+    this.importTransformer = importTransformer;
+  }
+
+  public static interface OutputImportTransformer {
```

Comment:
Add doc to this interface. In particular, clarify that this computes any imports necessary for the sample output to execute correctly, and which must be passed to the `MethodSampleView`